src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -382,10 +382,10 @@ private function formatPath($path, $line)
         if ($linkFormat = $this->fileLinkFormat) {
             $link = str_replace(array('%f', '%l'), array($path, $line), $linkFormat);
 
-            return sprintf(' in <a href="%s" title="Go to source">%s line %d</a>', $link, $file, $line);
+            return sprintf(' in <a href="%s" title="Go to source">%s line %d</a>', htmlspecialchars($link), htmlspecialchars($file), $line);
```

Comment:
The html escaping function to use is `$this->escapeHtml()` (see https://github.com/symfony/symfony/pull/14694/files#diff-facc3bcdbfaa88c930890ad4713b187dR379)
Btw, because $path is already escaped, you double-escape this part of the $link